File: Famoser.BeerCompanion.Data/Entities/DrinkerEntity.cs
using System;
using System.Collections.Generic;

namespace Famoser.BeerCompanion.Data.Entities
{
    public class DrinkerEntity
    {
        public Guid Guid { get; set; }

        public string Name { get; set; }
        public string Color { get; set; }

        public int TotalBeers { get; set; }
        public string LastBeer { private get; set; }

        public DateTime? LastBeerTime
        {
            get
            {
                DateTime res;
                if (DateTime.TryParse(LastBeer, out res))
                    return res;
                return null;
            }
        }

        public List<Guid> AuthDrinkerCycles { get; set; }
        public List<Guid> NonAuthDrinkerCycles { get; set; }
    }
}

File: Famoser.BeerCompanion.Webpage/models/entities/DrinkerEntity.php
<?php
/**
 * Created by PhpStorm.
 * User: Florian Moser
 * Date: 11.01.2016
 * Time: 12:36
 */

namespace famoser\beercompanion\webpage\models\entities;


use famoser\beercompanion\webpage\models\Drinker;

class DrinkerEntity extends Drinker
{
    public function __construct(Drinker $drinker)
    {
        //$this->Id = $drinker->Id; Id censored
        //$this->Guid = $drinker->Guid; Guid censored
        $this->Name = $drinker->Name;
        $this->Color = $drinker->Color;
        $this->LastBeer = $drinker->LastBeer;
        $this->TotalBeers = $drinker->TotalBeers;
    }

    public $AuthDrinkerCycles = array();
    public $NonAuthDrinkerCycles = array();
}